src/util/AbstractAPI.js

```diff
@@ -107,12 +107,18 @@ Ext.define('Jarvus.util.AbstractAPI', {
                 if (response.aborted === true) {
                     Ext.callback(options.abort, options.scope, [response]);
                 } else if (response.status == 401 || response.statusText.indexOf('Unauthorized') !== -1) {
-
+                    
```

Comment:
@themightychris Fixed, I did it right in GitHub but didn't notice the extra space until after the fact. Sorry about that.